### DISCUSSION TITLE
Compatibility with DH18ZA1

### DISCUSSION THREAD

**Discussion by mrbluebrett** [omerfaruk-aran/esphome_samsung_hvac_bus#27] [Questions]
Hi, i have a samsung duct AC model DH18ZA1, but it’s old and I’m not quite sure about compatibility with this project.

In the schematics i see the F1, F2, V1, V2, but with labels I think they are not the same thing. See them here: 
![d3fa96ae-b141-4646-8e49-3828879e30d8](https://github.com/lanwin/esphome_samsung_ac/assets/77754688/2e071026-0540-4c34-846e-15958b199647)

The remote control is attached to R1, R2, R3… and Fs and Vs are for other things…

i also attach a picture of the remote (KR-H50110) if that helps:
![84a3390f-ec7c-49f1-aa21-831893ce3d31](https://github.com/lanwin/esphome_samsung_ac/assets/77754688/205cc2fb-4912-4d91-ad3b-6a0c07f97b99)

Any help in making this model controllable with wifi module would be great.
Thanks in advance.

**Comment by lanwin** [comment]
Hi, its pretty hard to say if this device will work or not. I am not a hardware guy. The cheapest option would be to buy an USB to RS-485 device and try to capture some traffic. Than we can see if this old device also uses the non NASA protocol. If not, it will be difficult.
